scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/util/TestConfiguration.java

```diff
@@ -4,6 +4,6 @@
 
 @TestPropertySource(properties = { "scalingenginejob.reschedule.interval.millisecond=100",
 		"scalingenginejob.reschedule.maxcount=5", "autoscaler.scalingengine.url=http://localhost:8090",
-		"scalingengine.notification.reschedule.maxcount=2" })
+		"scalingengine.notification.reschedule.maxcount=2", "server.ssl.enabled=false" })
```

Comment:
Now since we implemented SSL using org.apache.http.client.HttpClient, the unit tests will fail, so is it ok to have a flag to specify enable/disable SSL. Also in general it would be good to have a flag because of adding SSL support using HTTPClient...it is too closed and will give issues if certificates are not there. What are your thoughts?